spark/dl/src/test/resources/tf/loadTest/inception_resnet_v2.py

```diff
@@ -0,0 +1,95 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import tensorflow as tf
+import numpy as np
+import os
+import slim.nets.inception_resnet_v2 as inception_resnet_v2
+
+import argparse
+import sys
+
+from google.protobuf import text_format
+
+from tensorflow.core.framework import graph_pb2
+from tensorflow.core.protobuf import saver_pb2
+from tensorflow.python.client import session
+from tensorflow.python.framework import graph_util
+from tensorflow.python.framework import importer
+from tensorflow.python.platform import app
+from tensorflow.python.platform import gfile
+from tensorflow.python.training import saver as saver_lib
+
+
+def main():
+    """
+    You can also run these commands manually to generate the pb file
+    1. git clone https://github.com/tensorflow/models.git
+    2. export PYTHONPATH=Path_to_your_model_folder, eg. /home/models/
+    3. python inception_v3.py
+    4. wget https://raw.githubusercontent.com/tensorflow/tensorflow/v1.0.0/tensorflow/python/tools/freeze_graph.py
+    5. python freeze_graph.py --input_graph model/inception_resnet_v2.pbtxt --input_checkpoint model/inception_resnet_v2.chkp --output_node_names="InceptionResnetV2/AuxLogits/Logits/BiasAdd,InceptionResnetV2/Logits/Logits/BiasAdd,output1,output2" --output_graph inception_resnet_v2_save.pb
+    """
+    dir = os.path.dirname(os.path.realpath(__file__))
+    if not os.path.isdir(dir + '/model'):
+        os.mkdir(dir + '/model')
+    batch_size = 5
+    height, width = 299, 299
+    num_classes = 1001
+    # inputs = tf.placeholder(tf.float32, [None, height, width, 3])
+    inputs = tf.Variable(tf.random_uniform((2, height, width, 3)), name='input')
+    net, end_points = inception_resnet_v2.inception_resnet_v2(inputs,is_training = False)
+    output1 = tf.Variable(tf.random_uniform(tf.shape(net)),name='output1')
+    result1 = tf.assign(output1,net)
+    output2 = tf.Variable(tf.random_uniform(tf.shape(end_points['AuxLogits'])),name='output2')
+    result2 = tf.assign(output2,end_points['AuxLogits'])
+    saver = tf.train.Saver()
+    with tf.Session() as sess:
+        init = tf.global_variables_initializer()
+        sess.run(init)
+        sess.run([result1,result2])
+        checkpointpath = saver.save(sess, dir + '/model/inception_resnet_v2.chkp')
+        tf.train.write_graph(sess.graph, dir + '/model', 'inception_resnet_v2.pbtxt')
+        tf.summary.FileWriter(dir + '/log', sess.graph)
+
+
+    input_graph = dir + "/model/inception_resnet_v2.pbtxt"    
+    input_checkpoint = dir + "/model/inception_resnet_v2.chkp"
+    output_node_names= "InceptionResnetV2/AuxLogits/Logits/BiasAdd,InceptionResnetV2/Logits/Logits/BiasAdd,output1,output2"
+    output_graph = dir + "/inception_resnet_v2_save.pb"
+    # default value do not need to change for different models
+    restore_op_name = "save/restore_all"
+    filename_tensor_name = "save/Const:0"
+
+    input_graph_def = graph_pb2.GraphDef()

```

Comment:
can you wrap this piece of code as a routine and use it in all module test files?